Resources/Prototypes/SS220/Catalog/Fills/Crates/engines.yml

```
@@ -0,0 +1,11 @@
+- type: entity
+  id: CrateEngineeringTEG
+  parent: CrateEngineeringSecure
+  name: Ящик с двигателем ТЭГ
```

```suggestion
  name: ящик с двигателем ТЭГ
```
Я точно не помню, но по идее, в товарах названия всё также с маленькой буквы идут, не?